src/mscorlib/src/System/IO/UnmanagedMemoryStream.cs

```diff
@@ -334,10 +325,6 @@ public override long Position
                     throw new NotSupportedException(SR.NotSupported_UmsSafeBuffer);
                 if (!_isOpen) __Error.StreamIsClosed();
 
-                // Note: subtracting pointers returns an Int64.  Working around
```

Comment:
That seems reasonable. I was under the impression that we would want to have the same behavior as netfx and wait to throw until the next operations on the UMS that operate on the invalid position. Actually throwing for a negative position is certainly more useful though.